### ISSUE TITLE
when will the pretrained models be released?

### ISSUE THREAD

**Issue by Barry-Liang** [tudelft-iv/SliceMatch#1]
Thank you for open-sourceing such a great work. Will you release the pretrained models soon? I am preparing to take advantage of this work on my project .

**Comment by TedLentsch** [comment]
Hi, sorry for the late reply. Recently I was busy with a different project, and as a consequence, we still need to upload the training and evaluation code. We are working on cleaning the code and will upload the clean code together with the pretrained weights when the code cleaning is done.

**Comment by duanxingyu19** [comment]
Hi, Will you release the pretrained models soon? It's 2025 now, pls when will you share your code and pre-training
weight. Thank you very much!
